common/workunit/workunit.cpp

```diff
@@ -2607,7 +2607,7 @@ cost_type aggregateCost(const IConstWorkUnit * wu, const char *scope, bool exclu
         cost_type totalCost = 0;
         for (it->first(); it->isValid(); )
         {
-            stat_type value = 0.0;
+            stat_type value = 0;
```

Comment:
This is unrelated to this Jira but a minor fix not worth creating in a separate jira.